### ISSUE TITLE
Reference point location text unclear

### ISSUE THREAD

**Issue by JosiahCarberry** [markn86/moodle-mod_customcert#17]
The help for the Reference point location states "This specifies which location of the element to be located at position X and position Y." However, the options available are "Top right", "center" and "top left". I was under the impression that one of the major points of this plugin is to allow fine tuned and custom positioning of any elements.

This is for version 2016052300 running on Moodle 3.1+ (Build: 20160603)

**Comment by markn86** [comment]
The position of the elements is done on the 'Rearrange elements' page (link under the list of elements). This reference point refers to where the element is rendered. For example, if you have the reference point set to the left and placed a 'Student name' element in the center of the 'Rearrange elements' page it would not always be centered. This is because students' names vary in length and won't always be the same as the example text on the 'Rearrange elements' page. So, in this example, you would put the reference point as the center so that the center of the student's name would be at the center of the page (where you placed it on the 'Rearrange elements' page).

I hope that makes sense.

**Comment by markn86** [comment]
I suppose the help dialogue could use more details. I just can't think of a good way of describing it (as evident above).

**Comment by JosiahCarberry** [comment]
First of all, I completely deleted, then reinstalled the plugin. The "rearrange elements" link now appears, whereas it did not previously. The only difference is that the first time around I downloaded the plugin from the moodle.org site, whereas the second time from github. I did not trouble myself to compare the two.
Secondly, I presume that the options of "center", "upper left" and "upper right" refer to the positioning of the element within a text box or block, whereas the rearrange elements allows you to position the text box itself. If that is a correct assumption, you might update the help text accordingly.
